scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -294,8 +294,8 @@ task IsVQSRLite {
   command {
     set +e
 
-    # See if there are any non-header lines that contain the string 'AS_VQS_SENS'. If so, grep will return 0 else 1
-    grep -v '^#' ~{input_vcf} | grep AS_VQS_SENS > /dev/null
+    # See if there are any non-header lines that contain the string 'CALIBRATION_SENSITIVITY'. If so, grep will return 0 else 1
+    grep -v '^#' ~{input_vcf} | grep CALIBRATION_SENSITIVITY > /dev/null
```

Comment:
nit--but why do we do this? to check that there is a filter / the filter we expect?

Reply:
Yes - this is how we determine if the supplied VCF was processed by VQSR Lite